docs/csharp/discards.md

```diff
@@ -34,7 +34,7 @@ The `Deconstruct` method of a class, structure, or interface also allows you to
 
 For more information on deconstructing user-defined types with discards, see [Deconstructing tuples and other types](deconstruct.md#deconstructing-a-user-defined-type-with-discards).
 
-## Pattern matching with `switch
+## Pattern matching with `switch`
```

Comment:
@DavidAnson Is there a markdownlint rule to catch this?

Reply:
@Youssef1313 No, but there is an ask for one: https://github.com/DavidAnson/markdownlint/issues/331